.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,7 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Bitcoin Core Developers
+    url: "https:bitcoincore.org"
```

Comment:
Shouldn't this be `https://bitcoincore.org`?

Reply:
Yep!

Reply:
Fixed in 6d67ee23dbbe007a910e7cebb0d3ee4d1268ca15

Reply:
Commit is now b18032cfa6ee590f231448e67e05c862750ea61f since I fixed the config.yaml not being detected (must be config.yml)